components/src/forms/LegacyInputField.tsx

```diff
@@ -57,7 +57,7 @@ export interface InputFieldProps {
  * @deprecated Use `InputField` in App/atoms instead
```

Comment:
can you modify this comment to remove the App/atoms? Since `InputField` will be moved in a follow up

```suggestion
 * @deprecated Use `InputField`
```